src/core/include/openvino/opsets/opset.hpp

```diff
@@ -24,8 +27,11 @@ class OPENVINO_API OpSet {
     static std::mutex& get_mutex();
 
 public:
-    OpSet() = default;
-    virtual ~OpSet() = default;
+    OpSet();
+    OpSet(const std::string& name);
+    OpSet(const ov::OpSet& opset);
+    OpSet operator=(const ov::OpSet& opset);
```

Comment:
Where do we need this operator?

Reply:
It's required here - https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/ir/src/input_model.cpp#L209